### PR TITLE
corpses don't shiver

### PR DESCRIPTION
**Pull request by BenLubar**
[MINOR]

## About the PR

Stops dead people from shivering if they're cold.

## Why's this needed?

Dr. Floorpills told me to do it.

### CODE REVIEW

**Comment by flrsh** [comment]
Small nitpick, can you move the `isalive(owner)` to the front?

File: code/mob/living/life/bodytemp.dm [hunk 1]
```diff
@@ -48,7 +48,7 @@
 			return ..()
 
 		//shivering status chance
-		if (owner.bodytemperature + owner.temp_tolerance < owner.base_body_temp && !owner.is_cold_resistant())
+		if (owner.bodytemperature + owner.temp_tolerance < owner.base_body_temp && !owner.is_cold_resistant() && isalive(owner))
```
Comment (ZeWaka) [inline code/mob/living/life/bodytemp.dm:51]: amb, grouping, parentheses please

Comment (pali6) [reply code/mob/living/life/bodytemp.dm:51]: it's just `&&` and basic operators, I think it's fairly unambiguous.

Comment (ZeWaka) [reply code/mob/living/life/bodytemp.dm]: specifically the addition and comparison

Comment (ZeWaka) [reply code/mob/living/life/bodytemp.dm]: it's even more ambiguous now

**Review by pali6 — APPROVED** [review]

**Review by ZeWaka — CHANGES_REQUESTED** [review]
previous review

**Review by Adharainspace — APPROVED** [review]

File: code/mob/living/life/bodytemp.dm [hunk 2]
```diff
@@ -48,7 +48,7 @@
 			return ..()
 
 		//shivering status chance
-		if (owner.bodytemperature + owner.temp_tolerance < owner.base_body_temp && !owner.is_cold_resistant())
+		if (isalive(owner) && (owner.bodytemperature + owner.temp_tolerance < owner.base_body_temp) && !owner.is_cold_resistant())
```
Comment (ZeWaka) [inline code/mob/living/life/bodytemp.dm:51]:
```suggestion
		if (isalive(owner) && ((owner.bodytemperature + owner.temp_tolerance) < owner.base_body_temp) && !owner.is_cold_resistant())
```

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**